backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt

```diff
@@ -2489,6 +2490,24 @@ internal class CodeGeneratorVisitor(val context: Context, val lifetimes: Map<IrE
     }
 }
 
+private fun IrValueParameter.nameConversion() = descriptor.name.nameConversion()
```

Comment:
`nameConversion` function name here and below is too ambiguous for this context.
Maybe `nameForDebugInfo` or `debugInfoName`?